gstd/gstd_daemon.c

```diff
@@ -72,6 +87,32 @@ gstd_daemon_init (gint argc, gchar * argv[], gchar * pidfilename)
   daemon_pid_file_proc = gstd_daemon_pid;
 
   _initialized = TRUE;
+  goto out;
+
+error:
+  {
+    switch (errno) {
```

Comment:
There is a function already to serialize the error: `strerror()`. We don't need this switch.

Reply:
Ok